pkg/mapper/dynamicfile/dynamicfile.go

```diff
@@ -21,13 +24,18 @@ type DynamicFileMapStore struct {
 	filename                  string
 	userIDStrict              bool
 	usernamePrefixReserveList []string
+
+	dynamicFileInitDone bool
 }
 
 type DynamicFileData struct {
+	// Time that the object takes from update time to load time
+	LastUpdatedDateTime string `json:"LastUpdatedDateTime"`
```

Comment:
This being a string is going to cause problems in the future. We have to migrate this to the right representation(epoch time is always int64) in a future change.

Reply:
Will take an AI, this requires changes across all components unfortunately